themes/theme-gmd/widgets/ImageSlider/spec.jsx

```diff
@@ -18,7 +18,7 @@ const createComponent = (props = {}) => mount(
   mockRenderOptions
 );
 
-describe('<ImageSliderWidget />', () => {
+describe.skip('<ImageSliderWidget />', () => {
```

Comment:
You can fix those tests by replacing `.find(SliderItem)` with `.find(SwiperItem)` and replacing `.find('div.sg-swiper-slide img')` with `.find('div.swiper-slide img')`.

Applying those changes made an issue visible for me. The `should render no slider with just a single image` test fails at the moment. I've checked some pages with sliders, and at e.g. the PDP you can slide when a product only has a single image.

Please take care, that one can't swipe when the Swiper only has a single slide.

Reply:
I took care of that all. Also, all ImageSliders now don't render a Swiper of there is only one Image. This also applies for the LiveShopping widget.